integrationtests/as-lucene-directory/src/test/java/org/infinispan/test/integration/as/wildfly/model/Member.java

```diff
@@ -27,7 +37,30 @@
    @GeneratedValue
    private Long id;
 
-   @Field(index = Index.YES, store = Store.NO)
+   @Latitude
+   Double latitude;
+
+   @Longitude
+   Double longitude;
```

Comment:
My rationale is that we have seen people going to the project integration tests to figure out how the functionality actually works and how to use it best and thus we should use the best practices. Though, the point about testing is good. But im still not on the breaking encapsulation boat. :)